mne/viz/_3d.py

```diff
@@ -416,6 +417,50 @@ def plot_evoked_field(evoked, surf_maps, time=None, time_label='t = %0.0f ms',
     return renderer.scene()
 
 
+def _get_meg_surf(meg, info, picks, trans, coord_frame, warn_meg):
+    from ..forward import _create_meg_coils
+    meg_rrs, meg_tris = list(), list()
+    coil_transs = [_loc_to_coil_trans(info['chs'][pick]['loc'])
+                   for pick in picks]
+    coils = _create_meg_coils(
+        [info['chs'][pick] for pick in picks], acc='normal')
+    offset = 0
+    for coil, coil_trans in zip(coils, coil_transs):
+        rrs, tris = _sensor_shape(coil)
+        rrs = apply_trans(coil_trans, rrs)
+        meg_rrs.append(rrs)
+        meg_tris.append(tris + offset)
+        offset += len(meg_rrs[-1])
+    if len(meg_rrs) == 0:
+        if warn_meg:
+            warn('MEG sensors not found. Cannot plot MEG locations.')
+    else:
+        meg_rrs = apply_trans(trans,
+                              np.concatenate(meg_rrs, axis=0))
+        meg_tris = np.concatenate(meg_tris, axis=0)
+    return dict(rr=meg_rrs, tris=meg_tris)
+
+
+def _get_coord_frame_trans(coord_frame, info, trans):
```

Comment:
This seems confusing, could you just transform to `head` if in "meg", then transform to `mri` if in originally in "meg" (now "head" because it was just transformed) or "head"? That seems like it might be more straightforward

Reply:
Ah I guess you might need them multiple times so that might not work